src/app/settings/admin/rule-groups/template.html

```diff
@@ -0,0 +1,176 @@
+        <div class="filters"
+             fxLayoutAlign=" center"
+             fxLayoutGap="16px">
+          <div>Filter by</div>
+          <mat-form-field fxFlex
+                          class="km-dropdown-without-subtext">
+            <mat-label>Seed</mat-label>
+            <mat-select disableOptionCentering
+                        class="km-select-ellipsis"
+                        [(ngModel)]="seedFilter"
+                        (selectionChange)="filter()">
+              <mat-select-trigger fxFlex
+                                  fxLayout="row">
+                <div fxFlex
+                     fxLayoutAlign="start">{{seedFilter}}</div>
+                <div fxFlex
+                     fxLayoutAlign="end"
+                     class="km-select-trigger-button-wrapper">
+                  <button *ngIf="seedFilter"
+                          (click)="seedFilter = undefined; filter(); $event.stopPropagation()"
+                          matSuffix
+                          mat-icon-button
+                          aria-label="Clear">
+                    <i class="km-icon-mask km-icon-remove"></i>
+                  </button>
+                </div>
+              </mat-select-trigger>
+              <mat-option *ngFor="let seed of seeds"
+                          [value]="seed">
+                {{seed}}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+          <mat-form-field fxFlex
+                          class="km-dropdown-without-subtext">
+            <mat-label>Type</mat-label>
+            <mat-select disableOptionCentering
+                        class="km-select-ellipsis"
+                        [(ngModel)]="typeFilter"
+                        (selectionChange)="filter()">
+              <mat-select-trigger fxFlex
+                                  fxLayout="row">
+                <div fxFlex
+                     fxLayoutAlign="start">{{typeFilter}}</div>
+                <div fxFlex
+                     fxLayoutAlign="end"
+                     class="km-select-trigger-button-wrapper">
+                  <button *ngIf="typeFilter"
+                          (click)="typeFilter = undefined; filter(); $event.stopPropagation()"
+                          matSuffix
+                          mat-icon-button
+                          aria-label="Clear">
+                    <i class="km-icon-mask km-icon-remove"></i>
+                  </button>
+                </div>
+              </mat-select-trigger>
+              <mat-option *ngFor="let type of ruleGroupTypes"
+                          [value]="type">
+                {{type}}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+        </div>
+        <div fxFlex></div>
+        <button mat-flat-button
+                color="quaternary"
+                fxLayoutAlign="center center"
+                (click)="add()">
+          <i class="km-icon-mask km-icon-add icon-add"></i>
```

Comment:
Are both `km-icon-add` and `icon-add` needed?

Reply:
For some reason the add icon is having too much margin on this button, so I needed to add some styling to it. I can also directly use `.km-icon-add` for it, but I thought we wanted to use the km-prefix only for global styling 😅

Reply:
True, but you can simply apply some additional styling on the component level to the global class. It will not override anything for other components.

Reply:
Alright. Then - changed :)